css-layouts/responsive-layouts/src/exercises/exercise2/index.js

```diff
@@ -0,0 +1,169 @@
+/*
+  ⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️⬇️
+
+  💡 Next, we're going to apply what we've learned about base-12 grids
+  💡 to arrange some content blocks. 
+
+  💡 The Block prop accepts either an integer representing a single width,
+  💡 or an object representing the widths specified at certain breakpoints.
+
+  💡 The breakpoints provided in the compoment are "small", "medium",
+  💡 and "large". These are intended to match up to mobile devices, 
+  💡 medium-range devices like tablets, and then desktops. 
+  
+  ℹ️ In practice, devices can have a large range of pixel sizes and 
+  ℹ️ layout changes might be needed at a larger variety of pixel widths. 
+
+  ⚠️ For example, if one were to pass in the following size object
+  ⚠️ to the Block component:
+
+  ```
+  size={{
+    small: 12
+    medium: 6
+    large: 4
+  }}
+  ```
+
+  💡 The computed CSS would set the width for that element to be 100%
+  💡 of the container at small range, half the width at medium range, 
+  💡 and one-third of the width of the page at large range. The "small"
+  💡 value represents the width that it should fall back to if a
+  💡 width isn't specified for a smaller size. It works the same as passing in
+  💡 only a single string value.
+
+  💡 Setting a Block component's width to the value of '0' will hide
+  💡 the element at that breakpoint. When developing a real-world application,
+  💡 you most likely would not want to hide a component using CSS for
+  💡 performance reasons. For the sake of this exercise, this prop is used 
```

Comment:
Might also be nice to add that hiding elements with CSS only is tricky for assistive technology, too!